spec/gogo/cmd/bnsd/x/account/codec.proto

```diff
@@ -1,33 +1,46 @@
 syntax = "proto3";
 
-package blueaccount;
+package account;
 
 import "codec.proto";
+import "coin/codec.proto";
 import "gogoproto/gogo.proto";
 
-// Domain represents a blue account domain.
+// Domain represents an account domain.
 //
 // Domain is a container for accounts. Account representation is
 // `<name>*<domain>`.
-// Each domain owner controls all accounts that belong to that domain.
-// Each domain controls expiration of all accounts that belong to that domain.
 //
-// Because of the functionality that the Red Account provides, Blue Account
-// domain must not be `iov` as this is a domain used by Red Account
-// functionality.
+// Each domain admin controls all accounts that belong to that domain.
+// Each domain controls expiration of all accounts that belong to that domain.
 message Domain {
   weave.Metadata metadata = 1;
   string domain = 2;
-  // Owner is a weave.Address that controls this domain and all accounts that
+  // Admin is a weave.Address that controls this domain and all accounts that
   // belong to this domain.
-  bytes owner = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
+  bytes admin = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
   // Valid until defines the expiration date for the domain. Expired domain
   // cannot be used or modified. None of the accounts that belong to an expired
   // domain can be used of modified as well.
   int64 valid_until = 4 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixTime"];
+  // Has Superuser is a feature switch flag.
+  bool has_superuser = 5;
+  // Msg fees declares an optional list of additional fees that paying is
+  // required when processing a message within this domain.
+  repeated MsgFee msg_fees = 6 [(gogoproto.nullable) = false];
+  // Account review defines the duration of the account renewal period for each
+  // account that belongs to this domain.
+  int64 account_renew = 7 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixDuration"];
+}
+
+// MsgFee defines a fee for processing a message of a given type, determined by
+// that message path.
+message MsgFee {
```

Comment:
`MsgFee` naming could be confusing since there are modules filled with msg fee functionality.

Reply:
done in https://github.com/iov-one/weave/pull/1071/commits/40fd491bc079aeabc58c6ba761d74f3a9bb9f9f6